package.json

```diff
@@ -10,7 +10,13 @@
   "engines": {
     "node": ">=18.14.1"
   },
+  "scripts": {
+    "build:cloudflare-pages": "yarn workspace @hono/vite-cloudflare-pages build",
+    "build:dev-server": "yarn workspace @hono/vite-dev-server build"
+  },
```

Comment:
We have to make `build` command because it will do `yarn build` in a GitHub release flow.

Reply:
@sor4chi 

How about using `yarn workspaces foreach` instead of `npm-run-all`.

https://v3.yarnpkg.com/cli/workspaces/foreach

Reply:
Wow, i didn't know the feature!
You mean we dont nessary to configure build:*?
We only configure `build: yarn workspaces foreach run build`, right?

Reply:
Yes. I think we don't need to build them in parallel, so we can use the `foreach` method of yarn.

Reply:
Okay, I fixed

Reply:
yes, we have to use berry option, right?

Reply:
I think `@yarnpkg/plugin-workspace-tools` is needed.

https://v3.yarnpkg.com/cli/workspaces/foreach